### PR TITLE
Css four column layout

### PR DESCRIPTION
**Pull request by randygerada**
Screenshot 2022-10-28 at 4 25 58 PM

### CODE REVIEW

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+<html>
+  <head>
+    <title>CSS Four Column Layout</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@100&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <h1>How It Works</h1>
+    <main class="screens">
+      <div class="screen-1">
```
Comment (BrettAlbright) [inline css-four-column-layout/index.html]: make sure that you are using proper columns here. They need to have a set width / flex basis

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looks good Randy, just needs a small update for the columns

**Comment by randygerada** [comment]
Screenshot 2022-10-30 at 5 49 44 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**